plugins/spu2-x/src/Windows/Spu2-X.rc

```diff
@@ -62,15 +44,14 @@ BEGIN
     LTEXT           "Module:",IDC_STATIC,161,16,50,9,NOT WS_GROUP
     LTEXT           "(speedup!) Skips reverb effects processing, but won't sound as good in most games.",IDC_STATIC,26,60,104,36
     LTEXT           "(currently requires manual configuration via the ini file)",IDC_STATIC,162,236,146,20
-    CTEXT           "100 ms (avg)",IDC_LATENCY_LABEL,224,86,58,9
+    CTEXT           "100 ms (avg)",IDC_LATENCY_LABEL,224,84,58,9
```

Comment:
Explain this change?

Reply:
It moves the "latency value" up so that it's aligned with the leftmost text. On the master branch, you'll notice that the text is slightly below where it should be.